common/achievement/evaluate.ts

```diff
@@ -25,7 +25,7 @@ async function _evaluateAchievement(
         where: {
             userId,
             metric: { in: metrics },
-            AND: relation ? { relation: { equals: relation } } : {},
+            relation: relation ? { startsWith: relation } : null,
```

Comment:
Not sure if null is right here. Maybe we should take all events, in case we don't have a relation?